src/models/index.js

```diff
@@ -64,6 +64,22 @@ fs
     }
   });
 
+const descriptiveDetails = () => {
+  const loggedUser = httpContext.get('loggedUser') ? httpContext.get('loggedUser') : null;
```

Comment:
These ternary statements can be simplified to just `||` statements.

i.e. `const loggedUser = httpContext.get('loggedUser') || null;` yields the same result but is more performant and easier to read.

Also, just a sort-of-nitpicky note, I think the "order" of the const declarations should match the order they are being called in the "return" statement below, but that's just an OCD-ish thing.